src/screens/epoch/components/epoch_detail/components/mobile/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import classnames from 'classnames';
+import {
+  Typography,
+  Divider,
+} from '@material-ui/core';
+import useTranslation from 'next-translate/useTranslation';
+import { EpochDetailType } from '../../../../types';
+import { useStyles } from './styles';
+
+const Mobile: React.FC<{epochDetail:EpochDetailType[] } & ComponentDefault> = (props) => {
+  const { t } = useTranslation('epoch');
+  const classes = useStyles();
+
+  return (
+    <div className={props.className}>
+      {props.epochDetail.map((x, i) => {
+        return (
+          <React.Fragment key={`${x.name}-${i}`}>
+            <div className={classes.root}>
+              <div className={classnames(props.className, classes.root)}>
```

Comment:
Seems redundant to have 2 div wrapper both with `classes.root`, is there a reason for this

Reply:
It was a mistake to make an extra layer, removed.